### PR TITLE
fix: update credential card title

### PR DESCRIPTION
**Pull request by wadeking98**
# Summary of Changes

Changed the credential card title to use the credential definition name instead of the schema name. If the credential definition name is `default` or `Default` then the credential card title will revert to using the schema name instead.

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/utils/helpers.ts [hunk 1]
```diff
@@ -12,6 +12,21 @@ import { parseUrl } from 'query-string'
 
 import { Attribute, Predicate } from '../types/record'
 
+export function parseCredDefName(credDefId?: string): string {
```
Comment (amanji) [inline core/App/utils/helpers.ts]: Can we organize/group functions by the info they extract? Ex. schema functions grouped with other schema functions, and likewise with cred defs.

**Review by amanji — CHANGES_REQUESTED** [review]

File: core/App/utils/helpers.ts [hunk 2]
```diff
@@ -12,6 +12,34 @@ import { parseUrl } from 'query-string'
...
```
Comment (amanji) [inline core/App/utils/helpers.ts]: I'm thinking this actually might be a good idea to pull the schema and cred def utility functions into their own module as well. They can then be imported and exported directly from this module.

Comment (wadeking98) [reply core/App/utils/helpers.ts]: change completed

File: core/App/utils/helpers.ts [hunk 3]
```diff
@@ -12,25 +11,16 @@ import { parseUrl } from 'query-string'
...
-    }
-  }
-  return { name, version }
-}
+import { parseCredDefName, credentialDefinition } from './cred-def'
+import { parseSchema, credentialSchema } from './schema'
 
-export function credentialSchema(credential: CredentialRecord): string | undefined {
-  return credential.metadata.get(CredentialMetadataKeys.IndyCredential)?.schemaId
+export function parsedCredDefName(credential: CredentialRecord): string {
```
Comment (amanji) [inline core/App/utils/helpers.ts]: Can also move these functions out.

Comment (wadeking98) [reply core/App/utils/helpers.ts]: change completed

**Review by amanji — APPROVED** [review]

**Comment by amanji** [comment]
LGTM! Can merge when ready

**Pull request merged**